lib/active_hash/base.rb

```diff
@@ -50,6 +50,10 @@ def field_names
         @field_names ||= []
       end
 
+      def column_names
```

Comment:
I imagine the point here is that we need strings and not symbols for whatever integration with need with the CSV gem? If so, can you please document that in an rdoc string above the method?